### DISCUSSION TITLE
deprecation of 🪳-infested 1.0.4

### DISCUSSION THREAD

**Discussion by vobu** [ui5-community/wdi5#389] [Announcements]
(duplicating the Twitter/Mastodon postoot here for reach)

⚠️ we had to deprecate 1.0.4 b/c of a nasty bug involving name resolution errors of npm modules.
👉 Please use <=1.0.3 or >1.0.4 (when published) 👈
Context: even though CI was 🟢 (https://github.com/ui5-community/wdi5/pull/382/checks), the end result is 💩
Sorry folks! We’ll up our CI game and fix this 🪳

https://twitter.com/_wdi5_/status/1602933770201489408?s=61&t=HFgk8Rob7TpJt1PDqgPuNQ

**Comment by brcontainer** [comment]
Hello @vobu

I'm trying to understand why I was mentioned in this issue. Sorry if the question sounds weird, but I must admit I'm confused by the notifications displaying this:

![image](https://user-images.githubusercontent.com/4368952/210297221-e4df14e8-adfa-45cf-9121-ee5d19e57349.png)

It is marked as `reason:mention`, could it be some bug in `/notifications`? Or really at some point was I mentioned?

Again I apologize for asking you. Thank you for your attention. 👍

**Comment by vobu** [comment]
there was a spam comment on this issue on Dec 29 that GH removed swiftly, but obviously not quick enough (since it reached your inbox). Apologies for the inconvenience! Nevertheless have a good start into this new year 🎆
